FMPhotoPicker/FMPhotoPicker/source/Utilities/FMLoadingView.swift

```diff
@@ -16,9 +16,14 @@ class FMLoadingView {
     static let shared = FMLoadingView()
     
     private init() {
-        let rootVC = (UIApplication.shared.delegate?.window??.rootViewController)!
+        if let rootVC = (UIApplication.shared.windows.first?.rootViewController) {
+            self.transparentView = UIView(frame: rootVC.view.frame)
+            
+        } else {
+            let windowFrame = UIApplication.shared.keyWindow?.frame
+            self.transparentView = UIView(frame: windowFrame ?? .zero)
+        }
```

Comment:
It's better to set the frame of `transparentView` inside the `show` function. Like this:
```
    func show() {
        guard let keyWindow = UIApplication.shared.keyWindow else { return }
        
        self.transparentView.frame = keyWindow.frame
        keyWindow.addSubview(self.transparentView)

        self.transparentView.alpha = 0
        self.indicator.startAnimating()
        UIView.animate(withDuration: kEnteringAnimationDuration, animations: {
            self.transparentView.alpha = 1
        })
    }
```
By doing so, you can completely remove the code that fetchs the `rootViewController`.

Reply:
working fine with me